server/blueprints/buySellBlueprint.py

```diff
@@ -0,0 +1,91 @@
+from sqlalchemy import select
+import sqlalchemy
+from sqlalchemy.orm import sessionmaker, Session
+import flask
+from auth.auth_tools import login_required
+from database.schema import Company, t_companyhistory
+# from plotly.offline import plot
+# from plotly.graph_objs import Scatter
+
+
+def create_blueprint(Makesession: sessionmaker):
+    buySellBlueprint = flask.Blueprint("buySellBlueprint", __name__)
+
+    @buySellBlueprint.route('/', methods=['GET'])
+    @login_required(Makesession)
+    def getStockDetails():
+
+        response_body = {
+            "error": None,
+            "data": None
+        }
+
+        session: Session = Makesession()
```

Comment:
Maybe create another python package and file and extract some of this functions functionality. It looks great. I'm more thinking from the perpsective of organization